config-windows.md

```diff
@@ -94,3 +94,42 @@ The following parameters can be specified:
         }
    }
 ```
+
+## <a name="configWindowsCredentialSpec" />Credential Spec
+
+You can configure a container's group Managed Service Accounts (gMSAs) via the OPTIONAL `credentialspec` field of the Windows configuration. For more information about gMSAs, see [Active Directory Service Accounts for Windows Containers][gMSAOverview]. For more information about tooling to generate a gMSA, see [Deployment Overview][gMSATooling]. 
```

Comment:
Travis [doesn't like the trailing whitespace here][1] (or after `"GroupManagedServiceAccounts": [` below).

[1]: https://travis-ci.org/opencontainers/runtime-spec/jobs/231727783#L355